semanticcpg/src/main/scala/io/shiftleft/semanticcpg/language/Location.scala

```diff
@@ -104,7 +104,18 @@ class LazyLocation(storedNode: StoredNode, filenameOverride: Option[String]) ext
 
   private lazy val method: Method = storedNode match {
     case cfgNode: CfgNode => cfgNode.method
-    case local: Local     => local.method.head
+    case _ => findParentMethod(storedNode).getOrElse {
+      throw IllegalStateException(s"Node not contained in a method $storedNode")
```

Comment:
We have nodes like `Member` or `DomNode` which just don't exist within a method. So this should be `.getOrElse(defaultString)` like with the other values.